pkg/common/constants/constants.go

```diff
@@ -50,6 +50,9 @@ const CPU = "vcore"
 const SparkLabelAppID = "spark-app-selector"
 const SparkLabelRole = "spark-role"
 const SparkLabelRoleDriver = "driver"
+const SparkDefaultUser = "root"
```

Comment:
Spark should not have a different user than YuniKorn. The default YuniKorn user is "nobody" - see constants.go.